.github/workflows/monorepo.yml

```
@@ -0,0 +1,61 @@
+name: Monorepo Identify Changes
+
+on:
+  pull_request:
+    types: [opened, synchronize] # Trigger on PR events
+    branches:
+      - main
+      - Dev
+      - Preview
+
+concurrency:
+    group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)